.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v3.2.0
+    rev: v4.4.0
```

Comment:
Any reason to do these bumps in this PR? I'm okay with it, just curious

Reply:
No not really. I had run `pre-commit autoupdate`

Initially, I only bumped `isort` because I wanted to know if the very minor differences with the `ruff` sorting rule were because of an old `isort` version.

I'll remove it so that the diff is cleaner.

Reply:
no real qualms here tbh will defer to you